app/build.gradle

```diff
@@ -47,33 +47,25 @@ android {
 
     buildTypes {
         debug {
-            if (localPropertiesFile.exists()) {
-                def localProperties = new Properties()
-                localProperties.load(new FileInputStream(localPropertiesFile))
-
-                if(localProperties.getProperty('youtubeApiKey') != null && !localProperties.getProperty('youtubeApiKey').isEmpty()){
-                    resValue("string", "youtubeApiKey", localProperties['youtubeApiKey'])
-                }
-                else{
-                    resValue("string", "youtubeApiKey", "test")
-                }
-
-                if (localProperties.getProperty('spotifyClientId') != null && !localProperties.getProperty('spotifyClientId').isEmpty()){
-                    resValue("string", "spotifyClientId", localProperties['spotifyClientId'])
-                }
-                else{
-                    resValue("string", "spotifyClientId", "test")
-                }
-            }
-            else{
-                resValue("string", "youtubeApiKey", "test")
-                resValue("string", "spotifyClientId", "test")
-            }
-            resValue("string", "sentryDsn", "")
+            addLocalProperties(debug, localPropertiesFile)
+
+            buildConfigField "String", "API_BASE_URL", '"https://api.listenbrainz.org/1/"'
+            buildConfigField "String", "WEBSITE_BASE_URL", '"https://listenbrainz.org/"'
 
             applicationIdSuffix ".debug"
             versionNameSuffix ".debug"
         }
+        beta {
+            debuggable true
+            addLocalProperties(beta, localPropertiesFile)
+
+            buildConfigField "String", "API_BASE_URL", '"https://beta-api.listenbrainz.org/1/"'
+            buildConfigField "String", "WEBSITE_BASE_URL", '"https://beta.listenbrainz.org/"'
+
+            applicationIdSuffix ".beta"
```

Comment:
This defeats the whole purpose of us making this separation. We need to create separate environments and not builds. We want to be able to select the beta configuration and deploy that app on the playstore for a beta. The app shouldn't say ListenBrainz.beta when downloaded from the playstore.

Reply:
If we are going to define a type under buildType {}, It will turn out as either one of these. We are not going to use this build anywhere other than local development. 

Usually it's recommended to go with productFlavour for this but that idea was trashed.

If you want to make releases using this build we can remove the beta suffix.